.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
please add .vscode folder in the .gitignore file. 
https://www.w3schools.com/git/git_ignore.asp?remote=github
you should not push editor config files.